src/main/java/uk/nhs/hee/tis/trainee/sync/mapper/TraineeDetailsMapper.java

```diff
@@ -41,7 +44,7 @@
 import uk.nhs.hee.tis.trainee.sync.mapper.util.TraineeDetailsUtil.Title;
 import uk.nhs.hee.tis.trainee.sync.model.Record;
 
-@Mapper(componentModel = "spring", uses = TraineeDetailsUtil.class)
+@Mapper(componentModel = "spring", uses = TraineeDetailsUtil.class, unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

Comment:
This line was flagged as a code smell for being too long.

Reply:
Hmm.  I thought it would be but wasn't picked up by the codestyle.